specs/paths/statements/qualifiers/single.json

```diff
@@ -27,7 +27,7 @@
         ],
         "requestBody": { "$ref": "../../../global/requests.json#/Qualifier" },
         "responses": {
-            "200": { "$ref": "../../../global/responses.json#/QualifierHash" },
+            "200": { "$ref": "../../../global/responses.json#/Snak" },
```

Comment:
I think that for the responses at least we need to include the hash prop (we can do that with model compositions), so that user will be awar that that's where they get qualifier / reference hashes from.

Reply:
Yes, I'll be writing that commit now :)

Reply:
Done in https://github.com/wmde/wikibase-rest-api-proposal/pull/63/commits/c2d9ede38b8e056e89e54abfb9441a721818991e